src/systems/gltf-animations.js

```diff
@@ -0,0 +1,31 @@
+var registerSystem = require('../core/system').registerSystem;
+var THREE = require('../lib/three');

```

Comment:
Components and systems should have the same name. So you can do ```this.system``` within the component.